playground/App.vue

```diff
@@ -84,7 +84,7 @@ const handelButtonClick = () => {
           </div>
         </CarouselSlide>
         <template #addons>
-          <CarouselPagination />
+          <CarouselPagination paginated />
```

Comment:
The name `paginated` is ambiguous. I suggest using a more descriptive name like `singleItemPerPage`

Reply:
Okay, we can go with `paginateByItemsToShow`